src/js/state.js

```diff
@@ -23,18 +23,32 @@ var Wand = (function(wand) {
     if (nodeId !== wand.state.getCurrentNode()) {
       _state.push(nodeId);
       history.pushState(
-        wand.state.getState(), '', addUrlParam(document.location.search, "wandState", encodeState(wand.state.getState()))
+        wand.state.getState(), '',
+        addUrlParam(document.location.search, "wandState", encodeState(wand.state.getState()))
       );
     }
   };
 
   function getStateFromUrl() {
+    var nodeId;
     var _loadState = decodeState(getParameterByName('wandState'));
+    _state = _loadState;
+
     if (_loadState === '') {
-      return Wand.opts.nodes[0].id;
+      resetState();
+      nodeId = Wand.opts.nodes[0].id;
+    } else if (!wand.util.getNodeObject(wand.state.getCurrentNode())) {
+      wand.notifications.add({
+        text: 'Invalid wand state. Directing you to the beginning instead.',
+        time: 3500,
+        type: 'warning'
+      });
+      resetState();
+      nodeId = Wand.opts.nodes[0].id;
+    } else {
+      nodeId = wand.state.getCurrentNode();
     }
-    _state = _loadState;
-    return wand.state.getCurrentNode();
+    return nodeId;
   }
 
   function resetState() {
```

Comment:
I don't think for now, but there might be some use for it later. If you need it to do work on #43 then sure.